sote/game/scenes/game.lua

```diff
@@ -812,16 +814,278 @@ function gam.draw()
 		end
 		gam.planet_shader:send('tile_neighbor_realm', gam.tile_neighbor_realm_texture)
 	end
-
 	--- example of usage of recalculate_smooth_data_map
-	--[[
-	if gam.planet_shader:hasUniform("tile_sea") then
-		if gam.DATA_TEXTURES_CACHE["sea"] == nil then
-			gam.recalculate_smooth_data_map(sea, "sea")
+
+	---commenting
+	---@param a Tile
+	---@param b Tile
+	local function sample_face(a, b)
```

Comment:
It's unclear what the function parameters are from the function name.
Could we rename them, provide a comment or rename the function instead?
Also, it's missing the type for the return value.